assets/js/atomic/blocks/product/category-list/style.scss

```diff
@@ -0,0 +1,23 @@
+.wc-block-layout .wc-block-components-product-category-list {
+	margin-top: 0;
+	margin-bottom: $gap-small;
+
+	ul {
+		margin: 0;
+		padding: 0;
+		list-style: none outside;
+		display: inline;
+
+		li {
+			display: inline;
```

Comment:
Same as before. :slightly_smiling_face:  I would move the `list-style` property under `li`.